paasta_tools/api/api.py

```diff
@@ -65,6 +65,8 @@ def make_app():
     config.include('pyramid_swagger')
     config.add_route('service.instance.status', '/v1/services/{service}/{instance}/status')
     config.add_route('service.list', '/v1/services/{service}')
+    config.add_route('service.autoscaler.get', '/v1/services/{service}/{instance}/autoscaler')
+    config.add_route('service.autoscaler.post', '/v1/services/{service}/{instance}/autoscaler/{desired_instances}')
```

Comment:
I think I'd expect this to consume the instance count in the body, rather than as a path in the URL?

so
```
curl -X POST -d '{"count": "1"}' paasta.api.com/v1/services/myservice/myinstance/autoscaler
```

(that might not be quite correct, but you get what I mean)

If I'm reading this correctly, you'll be POSTing to ``/services/myservice/myinstsance/autoscaler/3`` and no body?